Mac/Preferences/Accounts/AccountsAddViewController.swift

```diff
@@ -15,7 +15,7 @@ class AccountsAddViewController: NSViewController {
 	
 	private var accountsAddWindowController: NSWindowController?
 	
-	private let addableAccountTypes: [AccountType] = [.onMyMac, .feedbin, .freshRSS]
+	private let addableAccountTypes: [AccountType] = [.onMyMac, .feedbin, .freshRSS, .feedly]
```

Comment:
Revert this change before merging, too.

Reply:
After talking with @vincode-io, it seems there is no good reason to revert this change.